StarterProject/specs/homepage.spec.js

```diff
@@ -0,0 +1,64 @@
+import Page from '../pageobjects/page.js';
+import HomePage from '../pageobjects/homePage.js';
+import NewArrivals from '../pageobjects/page.js';
+import Common from '../pageobjects/common.js';
+import FeaturedProductPage from '../pageobjects/featuredProductPage.js';
+
+
+const chai = require('chai');
+const should = chai.should();
+const expect = chai.expect;
+
+const mainPage = new Page();
+const common = new Common();
+const home = new HomePage();
+const featuredProductPage = new FeaturedProductPage();
+
+
+describe('Viewing Home Page',() => {
+
+     before(() => {
+        mainPage.open("http://www.clubmonaco.ca/home/index.jsp");
```

Comment:
This can just be a `browser.url()` call I think, to simplify having a separate `Page` class